portfolio/src/main/webapp/script.js

```diff
@@ -12,7 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//fetching function
+// function getHelloContent(){
+//     fetch("/data").then(response => response.text()).then((data) =>{
+//         document.getElementById("server-data").innerText = data;
+//     })
+// }
+
+function getLocalTime(timestamp){
+    const date = new Date(timestamp);
+    const localDate = new Date(date.getTime()+date.getTimezoneOffset()*60*1000);
+
+    localDate.setHours(date.getHours() - date.getTimezoneOffset() / 60);
+
+    return localDate; 
+}
+
+function createCommentElement(commentJson){
+    const commentContainer = document.createElement('div');
+    commentContainer.classList.add("comment-container");
+    
+    const author = document.createElement("h5");
+    const date = document.createElement("h7");
+    const comment = document.createElement("p");
+
+    const authorText = document.createTextNode(commentJson.author); 
+    const dateText = document.createTextNode(getLocalTime(commentJson.timestamp)); 
```

Comment:
Comments are currently showing "invalid date", is it because we haven't added timestamp field to comment yet?

Reply:
Yes, current version on server-side branch supports timestamp as well as handling POST requests.